cdk_framework/EKS/package.json

```diff
@@ -26,13 +26,16 @@
     "typescript": "~3.9.7"
   },
   "dependencies": {
-    "@aws-cdk/aws-ec2": "^1.161.0",
-    "@aws-cdk/aws-eks": "^1.161.0",
-    "@aws-cdk/aws-iam": "^1.161.0",
-    "@aws-cdk/core": "^1.161.0",
+    "@aws-cdk/aws-ec2": "1.161.0",
+    "@aws-cdk/aws-eks": "1.161.0",
+    "@aws-cdk/aws-iam": "1.161.0",
+    "@aws-cdk/core": "1.161.0",
+    "ajv": "8.11.0",
+    "ajv-errors": "3.0.0",
```

Comment:
Why did you remove the carat from some dependencies?
https://github.com/npm/node-semver#caret-ranges-123-025-004

Please use carat in all dependencies unless you have a good reason for not doing it.

Reply:
I had an issue with my dependencies not lining up, where I was getting the error illustrated in this github post - https://stackoverflow.com/questions/66572044/how-to-resolve-aws-cdk-error-argument-of-type-function-is-not-assignable-to-p#:~:text=37,to%20avoid%20different - so I followed the solution given.

Reply:
Now everything works, so I'll try re-adding the up-carats